srcts/src/components/popover.ts

```diff
@@ -282,15 +286,10 @@ export class BslibPopover extends BslibElement {
   // when the popover is hidden, we're responsible for moving it back to this
   // element.
   private _restoreContent(): void {
-    const { tip } = this.bsPopover;
-    if (!tip) {
-      throw new Error(
-        "Failed to find the popover's DOM element. Please report this bug."
-      );
-    }
-    const body = tip.querySelector(".popover-body");
+    const el = this.bsPopoverEl;
```

Comment:
Do we unset this element at some point? Or do we just update it when a new popover is inserted?

Reply:
Ah yes, good point. Addressed in [c0b95a9](https://github.com/rstudio/bslib/pull/766/commits/c0b95a9dee5357e107fde0b315023e82e1a645ee)